servers/rendering/renderer_rd/forward_mobile/render_forward_mobile.h

```diff
@@ -210,6 +210,8 @@ class RenderForwardMobile : public RendererSceneRenderRD {
 		float normal_xform[12];
 		float pad[3];
 		float exposure_normalization;
+		float texture_size[2];
+		float pad2[2];
```

Comment:
```suggestion
		float texture_size[2];
		float exposure_normalization;
		float pad;
```
For future reference, this entire struct should be refactored to only be 12 floats. 

GLSL handles mat3s really awkwardly, so they take 12 floats. Which means we have 3 floats of secret padding. Since we only have 3 additional floats of data, we can shrink the size of the struct, but that's work for another day